### PR TITLE
Fix traefik helm release

### PR DESCRIPTION
**Pull request by zc-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-5953

### Change description ###
fix traefik hr for flux v2

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/admin/traefik2/traefik2.yaml [hunk 1]
```diff
@@ -8,9 +8,13 @@ metadata:
 spec:
   releaseName: traefik
   chart:
-    repository: https://helm.traefik.io/traefik
-    name: traefik
-    version: 10.6.0
+    spec:
+      chart: ./
```
Comment (timja) [inline apps/admin/traefik2/traefik2.yaml:12]: is this right?

Comment (zc-hmcts) [reply apps/admin/traefik2/traefik2.yaml:12]: think so https://github.com/traefik/traefik

Comment (timja) [reply apps/admin/traefik2/traefik2.yaml:12]: Where have you configured the helm chart location?

Comment (zc-hmcts) [reply apps/admin/traefik2/traefik2.yaml]: added helm repo file

**Review by timja — APPROVED** [review]

**Pull request merged**